readthedocs/projects/models.py

```diff
@@ -1581,6 +1581,7 @@ def add_features(sender, **kwargs):
     INDEX_FROM_HTML_FILES = 'index_from_html_files'
     DONT_CREATE_INDEX = 'dont_create_index'
     USE_NEW_PIP_RESOLVER = 'use_new_pip_resolver'
+    DONT_INSTALL_LATEST_PIP = 'dont_install_latest_pip'
```

Comment:
Instead of `_LATEST` in the name, I'd use one that communicates the resolver. Like, `DONT_USE_NEW_PIP_RESOLVER` that follow the same pattern than the above feature flag.

Reply:
This is so we can re-use this in the future

Reply:
Makes sense to me. I know this code will eventually be stale, but the name isn't a huge issue, and the comment in the code makes it clear for now at least.